tests/versioning_3_test.go

```diff
@@ -878,6 +877,403 @@ func (s *Versioning3Suite) testIndependentActivity(behavior enumspb.VersioningBe
 	s.verifyWorkflowVersioning(tvWf, behavior, tvWf.Deployment(), nil, nil)
 }
 
+func (s *Versioning3Suite) TestChildWorkflowInheritance_PinnedParent() {
+	s.testChildWorkflowInheritance_ExpectInherit(false, false, vbPinned)
+}
+
```

Comment:
would it make sense to add one of these tests doing
`s.testChildWorkflowInheritance_ExpectInherit(false, true, vbPinned)` to check that the override is inherited?

Reply:
yes

Reply:
done